import scala.Array
import scala.scalanative.unsafe._

object Test {

  def main(args: Array[String]): Unit = {
    throw new Exception
  }

}